handbook/quality/README.md

```diff
@@ -0,0 +1,146 @@
+
+Once the critical bug is confirmed, customer experience needs to ping both customers and the community to warn them. If CX is not available, the oncall engineer is responsible for doing this.
+If a quick fix workaround exists, that should be communicated as well for those who are already upgraded.
+
+### Measurement
+We will track the success of this process by observing the throughput of issues through the system and identifying where buildups (and therefore bottlenecks) are occurring. 
+The metrics are: Total # bugs open. Bugs in each state (inbox, acknowledged, reproduced). Each week these are tracked and shared in the weekly update by Charlie Chance.
+
+### Orphans
+Occasionally, bugs may get lost if, for example, a label is misapplied. Miscategorized issues may slip through the filters and languish in a grey zone. The “orphans” and “reproduced orphans” states exist to catch these issues. 
+Every week, the head of product is responsible for reviewing these two states to identify any that are not properly categorized in the process.
+
+## Appendix A: Bug states and filters
+
+### Inbox
+The bug has just come in. 
+
+If using the standard bug report, the bug is labeled “bug” and “reproduce” and not assigned to anyone and not on a board. [See on Github](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+label%3Abug+label%3A%3Areproduce+-project%3Afleetdm%2F37+-project%3Afleetdm%2F40+sort%3Aupdated-asc).
+
+### Acknowledged 
+QA has gone through the inbox and has accepted it as a bug to be reproduced. 
+
+QA assigns themselves and adds it to the Release board under “awaiting QA”. [See on Github](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+label%3Abug+label%3A%3Areproduce+-project%3Afleetdm%2F37+sort%3Aupdated-asc).
+
+### Reproduced
+QA has reproduced the issue successfully. It should now be transferred to engineering to work on. 
+
+Remove the “reproduce” label and add the label of the relevant team (#agent, #platform, #interface) and assign it to the relevant engineering manager (make your best guess as to which team – the EM will re-assign if they think it belongs to another team). Move it to “Ready” in the Release board. [See on Github](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+label%3Abug+-label%3A%3Areproduce+-project%3Afleetdm%2F37+project%3Afleetdm%2F40+-assignee%3Axpkoala+sort%3Aupdated-asc).
+
+### Orphans 
+Bugs which do not have the reproduce label and do not exist on the release board. This filter serves as a sanity check. There should be no bugs in this state, because it means this bug is likely to be forgotten by our process. [See on Github](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+sort%3Aupdated-asc+label%3Abug+-label%3A%3Areproduce+-project%3Afleetdm%2F37+-project%3Afleetdm%2F40+).
```

Comment:
```suggestion
Bugs which do not have the reproduce label and do not exist on the release board. This filter serves as a sanity check. There should be no bugs in this state, because it means this bug is likely to be forgotten by our process. [See on GitHub](https://github.com/fleetdm/fleet/issues?q=archived%3Afalse+org%3Afleetdm+is%3Aissue+is%3Aopen+sort%3Aupdated-asc+label%3Abug+-label%3A%3Areproduce+-project%3Afleetdm%2F37+-project%3Afleetdm%2F40+).
```